# 例外を補えない

* チェック例外が発生した時、メソッド内で例外を補えないことも可能補えなかった例外は、呼び出し関係を通りながら送出される
* 発生したチェック例外を補えないということは、補足と対処を呼び出し元に任せるということ
* そのようなメソッドは`例外を投げる(補えない)メソッド`として宣言しなければならない
* [SampleException04](SampleException04.txt)